packages/vue/src/components/primitives/amplify-button.vue

```diff
@@ -5,6 +5,8 @@ interface PropsInterface {
   size?: 'small' | 'medium' | 'large';
   variation?: 'primary' | 'default' | 'link';
   fontWeight?: 'normal' | 'bold' | 'lighter';
+  loading?: boolean | string;
+  disabled?: boolean | string;
```

Comment:
Since you added `disabled` as a prop here, it's no longer adding the `disabled` attribute via `$attrs` to the button. You'll need to explicitly set `:disabled=disabled` on the button to correct this.

Reply:
thanks good catch, `:disabled="disabled"` has been added onto the button